apps/web/tailwind.config.js

```diff
@@ -1 +1,5 @@
-module.exports = require("@calcom/config/tailwind-preset");
+const base = require("@calcom/config/tailwind-preset");
+module.exports = {
+  ...base,
+  content: [...base.content, "../../packages/ui/**/*.{js,ts,jsx,tsx}"],
+};
```

Comment:
This is the way I found to customize the `tailwing` config from the monorepo without having to bring the contents and plugins entirely to here as before.